roles/glusterfs/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Set gluster fs facts
+  set_fact:
+    glusterfs_quorum:  "{{ (groups['glusterfs_nodes']) | count | int }}"
```

Comment:
this is just a number of gluster hosts. For zookeeper quorum required >50% of servers